### ISSUE TITLE
Auth0.js crashes phone

### ISSUE THREAD

**Issue by calclavia** [auth0/auth0.js#168]
I tried using Auth0.js v7.0.2 with an Ionic 2 project. It works in the browser, but every time it is run on mobile, I receive the error "Something went wrong" with no additional information.

I then decided to roll back to previous Auth0.js version v6.8. Installing the app in my phone (Samsung S6) permanently damaged the System UI. The phone now keeps displaying "System UI has stopped" even after restart (must factory reset phone). This is replicable.

Tried:
```
	this.auth.login({
					popup: true,
					connection: this.providerID,
					authParams: {
						scope: "openid name offline_access",
						device: "Mobile device"
					}
				},
				//	(err, result) => {
				(err, profile, id_token, access_token, refreshToken) => {
```

**Comment by gnandretta** [comment]
Hi, could you provide us a sample project that replicates the problem? Also does this happen when installing the application? when starting it? or when attempting to login?

Thank you.

**Comment by calclavia** [comment]
Follow up: So I updated my phone's OS, and it no longer crashes with Auth0 v6.8.

I'll see if I can replicate it on an empty project. I remember this problem happened on two different projects.

**Comment by cosemansp** [comment]
I had the same problem. There is a fundamental bug in the index.js.
You can find a fix in https://github.com/cosemansp/auth0.js (see index.js - startHandler function)
I submitted a pull request

**Comment by gnandretta** [comment]
Thank you @cosemansp. I've added a question in the PR.

**Comment by gnandretta** [comment]
Fixed in #172 and #173
